spacy_annotator/__init__.py

```diff
@@ -1,2 +1,257 @@
-from spacy_annotator.pandas_annotations import annotate
-from spacy_annotator.list_annotations import annotate
\ No newline at end of file
+from IPython.display import clear_output, display, display_html
+from ipywidgets import Button, HTML, HBox, Text, Output, Layout
+import pandas as pd
+import spacy
+from spacy import displacy
+from spacy.matcher import PhraseMatcher
+from spacy.tokens import Span
+import warnings
+
+
+class Annotator:
+    """
+    Helper class for SpaCy based NER annotation.
+    Parameters
+    ----------
+    model (spacy model) = SpaCy model for pre-annotation, optional.
+    labels (list): list of named entity labels.
+    delimiter (str): delimiter to separate entities in annotator. Default: ',' (comma).
+    attr (str): include option to skip example while annotating. Default: True.
+    include_skip (bool):
+    """
+
+    def __init__(
+        self,
+        *,
+        model=None,
+        labels,
+        delimiter=",",
+        attr="LOWER",  # "ORTH"
+        include_skip=True,
+    ):
+        self.model = model
+        if self.model is not None:
+            self.nlp = model
+        else:
+            # TODO think of better solution for default?
+            self.nlp = spacy.load("en_core_web_sm")
```

Comment:
What if I want to annotate in a secure env? Does this mean it will look for the model regardless? I'd prefer an option where if I don't supply a model it just does the vanilla thing

Reply:
Good point. I'm new to SpaCy, but think 

```python
from spacy.lang.en import English
nlp = English()
```

is what we want. It's a model-less language pipeline if i understand correctly (see [code](https://github.com/explosion/spaCy/blob/master/spacy/lang/en/__init__.py))

Reply:
sounds good